### PR TITLE
refactor #5 moved active_trail_nodes function

### LINKED ISSUE #5: Move `BayesianModel.active_trail_nodes` method
Move BayesianModel.active_trail_nodes to DAG

### PR DESCRIPTION
**Pull request by agylling**
Fixes #5

### CODE REVIEW

File: pgmpy/base/DAG.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def active_trail_nodes(self, variables, observed=None):
 
         Examples
         --------
-        >>> from pgmpy.models import BayesianModel
+        >>> from pgmpy.base import DAG
         >>> student = BayesianModel()
```
Comment (mackeper) [inline pgmpy/base/DAG.py]: >>> student = BayesianModel()
Should be:
 >>> student = DAG()

Comment (agylling) [reply pgmpy/base/DAG.py]: Fixed

**Comment by gpihl** [comment]
Looks Good

**Review by gpihl — APPROVED** [review]
Looks good

**Pull request merged**